assets/sass/widgets/_widget-area.scss

```diff
@@ -0,0 +1,36 @@
+/**
+ * Widget Area styles.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-widget-area-header {
+	margin: 0 0 1rem 0;
+
+	.googlesitekit-widget-area-header__title {
+		color: $c-tertiary;
+		font-weight: 400;
+		margin: 0;
+	}
+
+	.googlesitekit-widget-area-header__subtitle {
+		color: $c-boulder;
+		font-family: $f-primary;
+		font-size: 0.75rem;
+		font-weight: 400;
+		line-height: 1.2;
+		margin: 0;
+	}
+}
```

Comment:
Let's add some styling here based on `googlesitekit-widget-area-${ style }`:
* `.googlesitekit-widget-area-composite .googlesitekit-widget-area-widgets` should receive the white background and box shadow.
* `.googlesitekit-widget-area-boxes .googlesitekit-widget` should receive the white background and box shadow.